### PR TITLE
docs: correct mermaid docs after upgrade marked

### PR DESCRIPTION
**Pull request by Koooooo-7**
## Summary

correct mermaid docs after upgrade marked, and mentioned its limitation.
We should adoption the mermaid to resolve the async issue in #1433 then.

## What kind of change does this PR introduce?

Docs
## For any code change,

- [x] Related documentation has been updated, if needed
- [ ] Related tests have been added or updated, if needed

## Does this PR introduce a breaking change?

Yes
No

## Tested in the following browsers:

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Review by paulhibbitts — COMMENTED** [review]
Hi @Koooooo-7 , thanks so much for identifying this issue! I've been using the Mermaid plugin (https://github.com/Leward/mermaid-docsify which seems to support v10) so I did not even know of any support of Mermaid was possible in Docsify itself🙂

Happy to help tweak the updated docs, here is a first pass:

```
## Limited Support of mermaid

!> Currently, docsify only supports mermaid version `v9.3.0` (as the async render in mermaid `v10.x` is not yet supported in docsify)
```

I also have a preview build with this change for you to see:
https://paulhibbitts.github.io/docsify-v5-preview/#/markdown?id=limited-support-of-mermaid

And tried an example Mermaid 9.x diagram🙂
https://paulhibbitts.github.io/docsify-v5-preview/#/test-5

File: docs/markdown.md [hunk 1]
```diff
@@ -31,23 +31,24 @@ window.$docsify = {
 
 ## Supports mermaid
 
+!> Currently, docsify doesn't support the async mermaid render ( the limited supported latest merimaid version is `v9.3.0`).
```
Comment (trusktr) [inline docs/markdown.md:34]: merimaid -> mermaid

Perhaps 

> the latest mermaid version supported is `v9.3.0`

**Review by trusktr — CHANGES_REQUESTED** [review]
Looks good, just typo and adjustment and let's merge

**Comment by trusktr** [comment]
Btw what happens with latest mermaid? Is there an error or something?

File: docs/markdown.md (hunk 1 above)
Comment (paulhibbitts) [reply docs/markdown.md:34]: Good eyes @trusktr 👀, thanks!

Comment (sy-records) [inline docs/markdown.md:34]:
```suggestion
!> Currently, docsify doesn't support the async mermaid render (the latest mermaid version supported is `v9.3.0`).
```

**Review by sy-records — APPROVED** [review]

**Comment by Koooooo-7** [comment]
> Btw what happens with latest mermaid? Is there an error or something?

The mermaid makes a breaking change that it only support the async render after > v9.3, but we use the `marked` in synchronization. Hence user can not simply replace the code render with async functions, it gonna get a wrong `Promised`.

**Review by paulhibbitts — APPROVED** [review]
LGTM, thank you!

**Pull request merged**